lib/bots/syncbot.ts

```diff
@@ -286,6 +287,7 @@ export class SyncBot extends ProcBot {
 					}
 					if (!threadId && _.includes(actions, MessengerAction.CreateThread)) {
 						logger.log(LogLevel.INFO, `---> Creating thread '${data.details.title}' on ${toText}.`);
+						logger.log(LogLevel.DEBUG, `---> Full message from above, '${data.details.text}'.`);
```

Comment:
Not sure if this logger is some external library, but it might worth defining info/debug/etc methods on it and invoke them directly instead of needing to reach out to another type (i.e. LogLevel) — it feels slightly cumbersome.

Reply:
Logger, and by extension LogLevel, are an internal utility class that really could be extended with this structure.  But!  It is used all over ProcBots, so would not be a trivial amount of work.  I therefore have created an issue to contain this thought.

Reply:
Yeah, it need not happen overnight. You might as well add the methods, start using them from now on, and whenever you hit upon code that passes LogLevel manually, change that too.

Reply:
(And by that I don't mean those methods should be added in *this* PR :)

Reply:
Fair enough, that does tone it down into a trivial enough amount of work, and is happening right now!